src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -7,7 +7,11 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Reflection;
```

Comment:
nit: some usings above are unnecessary

Reply:
I'm going to leave `using System;` itself there because I find its absence is often inconvenient, but I will remove the others.